src/core/mac/mac.cpp

```diff
@@ -1117,12 +1117,12 @@ void Mac::BeginTransmit(void)
     // Disable CCA for the last attempt
```

Comment:
Since we clearly change behavior from CCA to CSMA-CA, shouldn't we change OPENTHREAD_CONFIG_DISABLE_CCA_ON_LAST_ATTEMPT accordingly?

Reply:
Thanks for pointing that out.  I made some updates.  Please review.